packages/node_modules/@webex/widget-call-history/src/CallHistoryItem.tsx

```diff
@@ -112,7 +113,7 @@ export const CallHistoryItem = ({
       className={cssClasses}
       onPress={onPress}
       data-testid="call-history-item"
-      aria-label={`${(name)}, ${(phoneNumber && formatPhoneNumberForAnnouncement(phoneNumber))}, ${(callingSpecific ? callingSpecific + ',' : '')} ${isOutgoing ? t('outGoingCallText') : isMissed ? t('missedCallText') : t('incomingCallText')}, ${(formatDurationForAnnouncement(durationSeconds as number))}, ${(startTime && isLocaleGerman ? formatDateDDMMYYYY(startTime) : formatDate(startTime))}, ${(startTime && isLocaleGerman ? formatTimeToSupport24Hours(startTime) : formatTime(startTime))}, ${t('recentCallsFocusButton')}`}
```

Comment:
Doesn't it break for German users?

Reply:
We have already taken care of displaying the date for German users. This announcement is in accordance with IVY comments; 
we need to announce the date, for example, June 20th, 2024